go.mod

```diff
@@ -93,3 +94,5 @@ require (
 )
 
 replace github.com/openstack-k8s-operators/openstack-operator/apis => ./apis
+
+replace github.com/openstack-k8s-operators/nova-operator/api => github.com/gibizer/nova-operator/api v0.0.0-20221129170244-4e3123df52f6
```

Comment:
This depends on https://github.com/openstack-k8s-operators/nova-operator/pull/168